sklearn/covariance/tests/test_robust_covariance.py

```diff
@@ -56,10 +56,10 @@ def test_mcd_class_on_invalid_input():
 
 
 def launch_mcd_on_dataset(
-    n_samples, n_features, n_outliers, tol_loc, tol_cov, tol_support
+    n_samples, n_features, n_outliers, tol_loc, tol_cov, tol_support, seed
 ):
 
-    rand_gen = np.random.RandomState(0)
+    rand_gen = np.random.RandomState(seed)
```

Comment:
Let me try to use the global seed here as well

Reply:
Thanks. Looks successful.